web/cobrands/sass/_layout.scss

```diff
@@ -27,7 +27,7 @@ $mappage-with-actions-sidebar-width: 58em;
     display: block;
 }
 .mob-only {
-    display: none;
+    display: none !important;
```

Comment:
A comment explaining why this is here (specificity in 3rd party CSS or similar) would be good.